blosc/tests/test.rs

```diff
@@ -10,11 +10,6 @@ use std::collections::hash_map::DefaultHasher;
 use std::hash::{Hash, Hasher};
 use std::mem;
 
-#[test]
-fn test_invalid_compressor() {
-    assert!(Context::new().compressor(Compressor::Invalid).is_err())
```

Comment:
The purpose of this test is to achieve coverage of the C blosc_get_complib_info function, so you should leave it in.

Reply:
This segfaults on some versions of `blosc`